File: check_file_existence/check_file_existence.pl
use strict;
use File::Basename;
my $exit_file = dirname(__FILE__).'/a.txt';
my $not_exist_file = dirname(__FILE__).'/b.txt';
if(-f $exit_file) {
  print "Found: ".$exit_file."\n";
}
unless(-f $not_exist_file) {
  print "Not Found: ".$not_exist_file."\n";
}

File: check_file_existence/check_file_existence.php
<?php
$exist_file = dirname(__FILE__).'/a.txt';
$not_exist_file = dirname(__FILE__).'/b.txt';
if(file_exists($exist_file) ) {
  print "Found: ".$exist_file."\n";
}
if(!file_exists($not_exist_file)) {
  print "Not Found: ".$not_exist_file."\n";
}